.pre-commit-config.yaml

```diff
@@ -96,7 +96,7 @@ repos:
         files: ^(homeassistant|pylint)/.+\.py$
       - id: pylint
         name: pylint
-        entry: script/run-in-env.sh pylint -j 0 --ignore-missing-annotations=y
+        entry: script/run-in-env.sh pylint -j 0 --ignore-missing-annotations=y --extension-pkg-whitelist=rrdtool
```

Comment:
Please move this change to the pylint `extension-pkg-allow-list` in the pyproject.toml config file.